docs/source/azure/configuration/helmtutorial.rst

```diff
@@ -4,4 +4,194 @@ Tutorial - Helm Deployment
 ==========================
 
 This is a step-by-step tutorial for deploying the Azure Container Host in Azure Kubernetes Service 
-(AKS) with the provided helm chart and deployment script.
+(AKS) with the provided helm chart and deployment script. This tutorial makes use of the 
+`Azure CLI <https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest>`_
+and `helm <https://helm.sh/>`_. Make sure that both are installed and you are logged in to your 
+subscription.
+
+1. Create a Resource Group
+--------------------------
+
+This will hold all of the Azure resources created in this tutorial. You could use an existing 
+resource group too.
+
+.. code-block:: bash
+
+    az group create --name c7n-helm-tutorial --location westus2
+
+2. Create a Storage Account
+---------------------------
+
+This storage account will hold all of the blobs and queues used by the container host. You could 
+also use an existing resource here or multiple storage accounts.
+
+We will also create a blob container. One will host the uploaded policies, and the other will store
+the custodian output files of running policies.
+
+.. code-block:: bash
+
+    az storage account create --resource-group c7n-helm-tutorial --name c7nstorage
+
+    account_key=$(az storage account keys list --account-name c7nstorage --query "[0].value" --output tsv)
+    az storage container create --account-name c7nstorage --account-key $account_key --name c7n-helm-policies
```

Comment:
nit: do we really need account key here?

Reply:
yeah, you do when using azure cli to create blob containers at least.